plugins/woocommerce/client/blocks/assets/js/blocks/product-filters/inner-blocks/taxonomy-filter/utils.ts

```diff
@@ -0,0 +1,21 @@
+/**
+ * External dependencies
+ */
+import { getSetting } from '@woocommerce/settings';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import type { TaxonomyItem } from './types';
+
+const taxonomies = getSetting< TaxonomyItem[] >(
+	'filterableProductTaxonomies',
+	[]
+);
+
+export function getTaxonomyLabel( taxonomy: string ) {
+	const match = taxonomies.find( ( item ) => item.name === taxonomy );
+	if ( match ) return match.label;
```

Comment:
I'm surprised linter passed, I was pretty sure we're forcing brackets around `if` statement body! 😆 

```suggestion
	if ( match ) {
	    return match.label;
	}
```